pkg/controllers/configobservation/oauth/observe_tokenconfig_test.go

```diff
@@ -34,6 +34,52 @@ func TestObserveTokenConfig(t *testing.T) {
 			},
 			errors: []error{},
 		},
+		{
+			name: "max age 0 still means default max age",
+			config: &configv1.OAuth{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cluster",
+				},
+				Spec: configv1.OAuthSpec{
+					TokenConfig: configv1.TokenConfig{
+						AccessTokenMaxAgeSeconds: 0,
+					},
+				},
+			},
+			previouslyObservedConfig: map[string]interface{}{},
+			expected: map[string]interface{}{
+				"oauthConfig": map[string]interface{}{
+					"tokenConfig": map[string]interface{}{
+						"accessTokenMaxAgeSeconds":    float64(86400),
```

Comment:
(No action required) Maybe this is just my ignorance of accepted practice for cluster operators, but I find it strange to see magic numbers rather than constants.